### ISSUE TITLE
UHC conditions aren't populating footer link to AHR

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1588]
**Describe the bug**
Each card has a footer listing the stat sources used in that card's calculations. Currently the UHC conditions like Vote Participation are only showing the footer link to ACS but not Americas Health Rankings 

**To Reproduce**
1. Go to "Voter Participation"
2. Look at the footer of each card

**Expected behavior**
Footer should show both ACS and AhR links

**Screenshots**

![image](https://user-images.githubusercontent.com/41567007/167977732-c2a6bad8-d760-4810-96b8-ddfc6df93711.png)

**Comment by joshzarrabi** [comment]
Good catch. I could have sworn I fixed this a while ago. I guess it broke again.
